sources/api/schnauzer/src/helpers.rs

```diff
@@ -1369,6 +1372,62 @@ enum OciSpecSection {
     ResourceLimits,
 }
 
+enum Runtime {
+    Docker,
+    Containerd,
+}
+
+impl Runtime {
+    /// Formats capabilities for different runtimes
+    fn get_capabilities(&self, caps: String) -> String {
+        match self {
+            Self::Docker => format!(
+                concat!(r#"["#, "{capabilities}", "],\n",),
+                capabilities = caps,
+            ),
```

Comment:
Per my other suggestion, I think it'd be better to have separate structs with different impls.